Fix misaligned block in UG

yucheng11122017 opened the pull request:
**What is the purpose of this pull request?**

- [X] Documentation update
- [X] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

<!--
  If this pull request is addressing an issue, link to the issue: #xxx

  If this pull request completely addresses an issue, use one of the closing keywords: "Fixes #xxx" or "Resolves #xxx"
  https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

  Otherwise, elaborate further on the rationale of this pull request as needed
-->

**Overview of changes:**
Closes #2124
Before: 
![image](https://user-images.githubusercontent.com/77204346/215999128-00ee3e66-93fe-4237-9e22-d102cdb0da36.png)

After: 
![image](https://user-images.githubusercontent.com/77204346/215998942-3b9b63da-9428-4d8e-a04d-9439f6f96252.png)



**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**
nil

<!--
  Any special testing instructions **not including** our automated tests.
-->

**Proposed commit message: (wrap lines at 72 characters)**
```
Fix misaligned block in UG
```
<!--
  See this link for more info on how to write a good commit message:
  https://se-education.org/guides/conventions/git.html

  As best as possible, write a succinct commit title in 50 characters

|---------This is the width of 50 chars----------|
|-----------This is the width of 72 chars for your reference-----------|
-->

---

**Checklist:** :ballot_box_with_check:

<!-- Leave non-applicable items unchecked -->

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [X] Linked all related issues
- [X] No unrelated changes <!-- It's tempting, but increases the reviewer's work, and really pollutes the commit history =( -->

<!--
  We'll try our best to get to your PR within a week.
  If we haven't gotten to it then, or if your pull request resolves an urgent item, feel free to give us a ping!
-->

tlylt commented on docs/index.md:
```diff
@@ -33,8 +33,8 @@ MarkBind extensions to Markdown | `==highlighted text==`<br>`%%grey text%%`<br>`
 <table>
-<tr>
-  <td>
+<tr >
```
is this empty space intentional? If so could you explain why this is needed ❓

tlylt reviewed (commented):
Thanks @yucheng11122017 for discovering and fixing this issue, I have never realised it.

Looking at the preview, the second pair of boxes still feels misaligned due to the difference in height, could we do something about it? (perhaps some inline CSS)
![image](https://user-images.githubusercontent.com/41845017/216058816-a32ab4b9-afc8-4c26-9654-81028046ba9b.png)

yucheng11122017 commented:
> Thanks @yucheng11122017 for discovering and fixing this issue, I have never realised it.
> 
> Looking at the preview, the second pair of boxes still feels misaligned due to the difference in height, could we do something about it? (perhaps some inline CSS) ![image](https://user-images.githubusercontent.com/41845017/216058816-a32ab4b9-afc8-4c26-9654-81028046ba9b.png)

Hi @tlylt is this better? 
![image](https://user-images.githubusercontent.com/77204346/216071553-ad918ddb-6c39-45fe-bb31-231737a874fc.png)

tlylt commented:
> Hi @tlylt is this better? ![image](https://user-images.githubusercontent.com/77204346/216071553-ad918ddb-6c39-45fe-bb31-231737a874fc.png)

Yes much better. Since we are fixing this part, could we perhaps make sure that this part looks nice on mobile as well? Currently, it is unscrollable and part of the content is hidden.
![image](https://user-images.githubusercontent.com/41845017/216200547-80b867b5-e5a3-4891-8bb5-16cf1716776d.png)

yucheng11122017 commented:
Hi @tlylt, two changes 
1. Added scrolling when width of screen is too small 
On mobile: 
> ![image](https://user-images.githubusercontent.com/77204346/216826163-9cf4d2d6-82eb-4a88-90dd-edf8182cc6b8.png)

2. Added codeWrapPlugIn so that user can wrap the long code if their screen is too small
On mobile: 
> ![image](https://user-images.githubusercontent.com/77204346/216826136-9b088a58-21b6-4191-917e-47e179988066.png)

This is when there is a bigger screen: 
> ![image](https://user-images.githubusercontent.com/77204346/216826202-e1aa888f-2726-4f68-929c-6bb0ec7c41af.png)

Hope these are ok!

tlylt commented:
Hi @yucheng11122017 
This is your bug report:
![image](https://user-images.githubusercontent.com/41845017/216830109-ef033bf2-4e85-4b4e-8375-3e56bedd8020.png)
This is the current result:
![image](https://user-images.githubusercontent.com/41845017/216830127-575df9e7-98de-454f-8fd9-6423f90d139f.png)
While slightly better, could we perhaps fix it the same way as the second example so that they are better aligned.

Also a side note, I am not sure why the `tip-box` has been changed to `box`, is this a related change required by this PR or due to the deprecation work in another PR?

yucheng11122017 commented:
> This is the current result: ![image](https://user-images.githubusercontent.com/41845017/216830127-575df9e7-98de-454f-8fd9-6423f90d139f.png) While slightly better, could we perhaps fix it the same way as the second example so that they are better aligned.

Hi @tlylt, sorry about this! I've changed it back to the second example.

tlylt commented on docs/index.md:
```diff
@@ -46,9 +48,8 @@ An example that uses GFMD syntax for task lists:
-  <td><br>
+  <td>
 <box>
-
```
This line needs to be retained in order for the `**Things to do:**` to be rendered as markdown, see now:
![image](https://user-images.githubusercontent.com/41845017/218233424-cf2881a0-2974-4550-8a18-fa963812f760.png)
vs before:
![image](https://user-images.githubusercontent.com/41845017/218233442-36d60cfb-f257-467f-8cd4-6b52d007851c.png)

Reason: https://markbind.org/userGuide/usingHtmlJavaScriptCss.html#markdown-in-html

yucheng11122017 replied:
Fixed in 2050515

tlylt replied:
Have you taken a look at the preview?
![image](https://user-images.githubusercontent.com/41845017/218233947-c8b1af9d-16b6-4c58-b570-ceb806c58373.png)

Please correct me if I'm seeing things wrong: the first example has a different alignment as compared to the second example. The first one is slightly misaligned and the second one is center-aligned. Are we able to make them align in the same way?

yucheng11122017 replied:
Ah sorry I thought that was fine based on our previous conversation. 

Computer preview: 
> ![image](https://user-images.githubusercontent.com/77204346/218265818-98159f45-a910-4ac7-b4e7-9effe9dcb6bc.png)

Mobile preview:
> ![image](https://user-images.githubusercontent.com/77204346/218265848-39880416-f488-4180-a715-435af841e82a.png)
